src/Microsoft.Azure.WebJobs.Host/Hosting/OptionsFormatter/IOptionsFormatter.Generic.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+
+namespace Microsoft.Azure.WebJobs.Hosting
+{
+    /// <summary>
+    /// Allows framework types to be registered as LoggableTypes.
```

Comment:
Is this comment out of date? What is a "LoggableType"?

Reply:
Yeah, out of date. Will update.